js/client-teambuilder.js

```diff
@@ -109,6 +109,7 @@
 		update: function () {
 			teams = Storage.teams;
 			if (this.curTeam) {
+				this.ignoreEVLimits = (this.curTeam.gen < 3) || (this.curTeam.format === 'classichackmons');
```

Comment:
You can remove the Classic Hackmons as it hasn't existed for a very long time, and won't be available in the future.